spotify_player/src/event/window.rs

```diff
@@ -164,12 +164,14 @@ pub fn handle_command_for_track_table_window(
             client_pub.send(ClientRequest::Player(PlayerRequest::StartPlayback(
                 base_playback
                     .uri_offset(tracks[id].id.uri(), state.app_config.tracks_playback_limit),
+                    false
```

Comment:
We should not change the shuffle state of the playback here, either from `false->true` or the opposite, as this behavior is unexpected from the user's perspective.

The `StartPlayback` should take `Option<bool>` instead, and the parameter here is `None`.